reference/best_practises/common_best_practises.rst

```diff
@@ -0,0 +1,26 @@
+Common Best Practises
+=====================
+
+The Akeneo PIM standard project comes with a default configuration to help you during the development of your project.
+
+
+What do you have to version?
+----------------------------
+
+* Do **NOT** version vendors
+* Do **NOT** modify the code in the vendors
+
+You will not benefit of dependencies patches otherwise.
+
+
+What about your dependencies?
+-----------------------------
+
+* You **MUST** update CE if you update EE
+* You **MUST** update EE if you update CE
+
+CE and EE versions are linked so you need to update both.
```

Comment:
That's part of our training dude! :cry: 
We don't know the side effects using CE 1.4.0 and EE 1.4.20. There are tested on the same version number.
Will say "If you're using Enterprise Edition, you MUST update Community Edition AND Enterprise Edition"